R/tt_pos_and_access.R

```diff
@@ -493,16 +564,19 @@ setGeneric("subset_cols",
                     newcinfo = NULL,
                     keep_topleft = TRUE,
                     keep_titles = TRUE,
+                    keep_fnotes = TRUE,
```

Comment:
I don't really see the need for this, if referential footnotes are on elements that remain after the subsetting, they should always be kept (but may need to be reindexed).

Can you tell me more about why you felt this should be added?

Unless this means _footer materials_, in which case I'm still not convinced why we would need a separate argument beyond titles, but I'm open to being convinced (if so the name will need to change though, fnotes in  a function/argument name means _referential footnotes_ in `rtables`)

Reply:
I intended general footers (main and prov). I thought it was giving another option to the user in case they need to keep only one of the two. Even if it is not yet used I think it is natural to ask if you can keep only titles or only footers when subsetting. I will change this into `keep_footers` for now. Let me know what you think ^^